staff/templates/staff_base.html

```diff
@@ -1,82 +1,211 @@
-      <li><a href="{% url 'classroom_management' %}">Manage Classrooms</a></li>
-      <li><a href="{% url 'form_overview' %}">Manage Forms</a></li>
-      <li><a href="{% url 'attendance' %}">Attendance</a></li>
-      <li><a href="{% url 'home-logout' %}">Logout</a></li>
-    </ul>
-  </div>
-</nav>
+            $sidedrawerEl.appendTo($overlayEl);
+            setTimeout(function() {
+              $sidedrawerEl.addClass('active');
+            }, 20);
+          }
 
 
-<body>
+          function hideSidedrawer() {
+            $bodyEl.toggleClass('hide-sidedrawer');
+          }
 
-{% block content %}
 
+          $('.js-show-sidedrawer').on('click', showSidedrawer);
+          $('.js-hide-sidedrawer').on('click', hideSidedrawer);
+        });
+    </script>
+    <script>
+    $(document).ready(function(){
+      $(".intro").click(function(){
+        if ($(this).hasClass("intro")){
+            $(this).removeClass("intro").addClass("deintro");
+        }else if($(this).hasClass("deintro")){
+            $(this).removeClass("deintro").addClass("intro");
+        }
+      });
+    });
 
-{% endblock %}
+    $(document).ready(function(){
+      var $rightpanel = $('#mui-panel');
+      $(".instafeedtrigger").click(function() {
+        if ($rightpanel.hasClass('rightpanelbody-hidden')) {
+            $rightpanel.removeClass('rightpanelbody-hidden').addClass('rightpanelbody-visible');
+        } else if ($rightpanel.hasClass('rightpanelbody-visible')){
+            $rightpanel.removeClass('rightpanelbody-visible').addClass('rightpanelbody-hidden');
+        };
+      });
+    });
+    </script>
+</head>
+<body class="hide-sidedrawer">
+    <header id="header" style="box-shadow: 0px 3px 35px 0px gray">
+      <div class="mui-appbar mui--appbar-line-height">
+        <div class="mui-container-fluid">
+          <a class="sidedrawer-toggle mui--visible-xs-inline-block mui--visible-sm-inline-block js-show-sidedrawer">☰</a>
+          <a class="sidedrawer-toggle mui--hidden-xs mui--hidden-sm js-hide-sidedrawer">☰</a>
+           <div style="float: right; cursor: pointer;" class="mui-dropdown mui-dropdown--left">
+              <a class="mui--text-left" data-mui-toggle="dropdown">
+                <i class="material-icons" style="color: white">more_vert</i>
+                <span class="mui-caret--left"></span>
+              </a>
+              <ul class="mui-dropdown__menu">
+                <li><a style="cursor: pointer;" class="instafeedtrigger">Instafeed</a></li>
+                <li><a href="{% url 'home-logout' %}">Log out</a></li>
+              </ul>
+          </div>
+        </div>
+        </div>
+      </div>
+    </header>
+    <div id="mui-panel" class="rightpanel rightpanelbody-hidden">
+      <h2 style="text-align: center; padding-bottom: 10px; font-size: 35px; color: #474747;">Instagram Feed</h2><hr style="width: 90%; height:2px;border:none;color:gray;background-color:gray; opacity: 0.2" />
```

Comment:
What is the use case for an instagram feed in this tool?

Reply:
No idea, it was suggested to me by our CEO

Reply:
We moved it